sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1145,6 +1136,16 @@ public abstract partial class ObjectSerializer
         public abstract System.Threading.Tasks.ValueTask SerializeAsync(System.IO.Stream stream, object? value, System.Type inputType, System.Threading.CancellationToken cancellationToken);
         public virtual System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public enum PropertyNamingConvention
+    {
+        None = 0,
+        CamelCase = 1,
```

Comment:
I feel like we explored the flags enum approach, and it had a number of negatives.  This is the approach I'd like to take because I think it sets us up best to evolve the type in response to customer feedback.  It follows precedents set in a variety of other .NET APIs, for databases and serialization, so I have higher confidence that it will be both consistent and evolvable over other approaches we've considered.